rpm/podman.spec

```diff
@@ -186,6 +186,16 @@ capabilities specified in user quadlets.
 It is a symlink to %{_bindir}/%{name} and execs into the `%{name}sh` container
 when `%{_bindir}/%{name}sh` is set as a login shell or set as os.Args[0].
 
+%define summary_machine Metapackage to install dependencies for %{name} machine
+
+%package machine
+Summary: %{summary_machine}
+Requires: %{name} = %{epoch}:%{version}-%{release}
+Requires: virtiofsd
```

Comment:
This should also add qemu and remove the Recommends: gvisor-tap-vsock and make it a requires here as this is also only used for mahcine

Reply:
done